pages/1.11/installing/oss/upgrading/index.md

```diff
@@ -17,14 +17,14 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 - Virtual networks require minimum Docker version 1.11. For more information, see the [documentation](/1.11/networking/virtual-networks/).
 - The DC/OS UI and APIs may be inconsistent or unavailable while masters are being upgraded. Avoid using them until all masters have been upgraded and have rejoined the cluster. You can monitor the health of a master during an upgrade by watching Exhibitor on port 8181.
 - Task history in the Mesos UI will not persist through the upgrade.
-- The latest version of Marathon-LB is required for 1.10. Before upgrading to 1.10, uninstall your existing Marathon-LB package and reinstall the updated version.
-- DC/OS 1.10 upgrades REX-Ray from v03.3. to v0.9.0 and therefore the REX-Ray configuration format has changed. If you have specified custom REX-Ray configuration in the `rexray_config` parameter of your `config.yaml` file, change the parameter to `rexray_config_preset: aws`.
+- The latest version of Marathon-LB is required for 1.11. Before upgrading to 1.11, uninstall your existing Marathon-LB package and reinstall the updated version.
+- DC/OS 1.11 upgrades REX-Ray from v03.3. to v0.9.0 and therefore the REX-Ray configuration format has changed. If you have specified custom REX-Ray configuration in the `rexray_config` parameter of your `config.yaml` file, change the parameter to `rexray_config_preset: aws`.
```

Comment:
This shouldn't be in the 1.11 docs. The upgrade from REX-Ray v0.3.3 to v0.9.0 was performed and completed in DC/OS 1.10.